tests/test_internals/test_tools.py

```diff
@@ -181,3 +182,34 @@ def tearDown(self):
 
     def test_dandi_upload(self):
         dandi_upload(dandiset_id="200560", nwb_folder_path=self.nwb_folder_path, staging=True)
+
+
+@pytest.mark.skipif(
+    not (HAVE_GLOBUS and LOGGED_INTO_GLOBUS),
+    reason="You must have globus installed and be logged in to run this test!",
+)
+class TestGlobusTransferContent(TestCase):
+    def setUp(self):
+        self.tmpdir = Path(mkdtemp())
+
+    def tearDown(self):
+        rmtree(self.tmpdir)
+
+    def test_transfer_globus_content(self):
+        """Test is fixed to a subpath that is somewhat unlikely to change in the future."""
+        source_id = "188a6110-96db-11eb-b7a9-f57b2d55370d"  # Buzsaki
+        destination_id = deploy_process(command="globus endpoint local-id", catch_output=True)
+        test_source_files = [
+            ["/PeyracheA/Mouse12/Mouse12-120815/Mouse12-120815.clu.1"],
+            [f"/PeyracheA/Mouse12/Mouse12-120815/Mouse12-120815.clu.{x}" for x in range(2, 4)],
+            [f"/PeyracheA/Mouse12/Mouse12-120815/Mouse12-120815.clu.{x}" for x in range(4, 9)],
+        ]
+        success, task_ids = transfer_globus_content(
+            source_id=source_id,
+            source_files=test_source_files,
+            destination_id=destination_id,
+            destination_folder=self.tmpdir,
+            display_progress=False,
+        )
+        assert success
+        assert task_ids
```

Comment:
Is this just testing that the function does not return empty arguments? Can we test size file in the temporary folder?

Reply:
`success` is `bool`, so asserts that it is True.

`task_ids` is a list, so asserts that it is not empty.

> Can we test size file in the temporary folder?

Yeah, sure.

Reply:
Thanks for pointing this out, actually revealed an issue with the relative globus paths. Working on it now.